woof-code/rootfs-skeleton/usr/sbin/puppyinstaller

```diff
@@ -3,14 +3,16 @@
 #(c) Copyright 01micko, shinobar and others
 #2007 Lesser GPL licence v2 (/usr/share/doc/legal/lgpl-2.1.txt)
 #140705 major overhaul of guis and conditions.
+#2023 a few changes made to correct some misinformation. 
+
 
 # to enable full install to hdd (before running this script):
 #   export ENABLE_FULLINSTALL=1 
 
 [ "`whoami`" != "root" ] && exec sudo -A ${0} ${@} #110505
 
 if [ -d /sys/firmware/efi ] ; then
-	/usr/lib/gtkdialog/box_yesno --ok-cancel --no-first --yes-label Continue --no-label Close --warning "Puppy Installer" "This installer does not support UEFI. It's only useful if you want to install to a drive that will not be booted on this machine.."
+	/usr/lib/gtkdialog/box_yesno --ok-cancel --no-first --yes-label Continue --no-label Close --warning "Puppy Installer" "This installer is ment to be used for Hard drive installs. It is known to cause errors in vanilla Dpup and other pups with a BDRV file, Check your USB ( or other istall media) to see if one is present."
```

Comment:
Maybe just fix the installer and copy *.sfs instead of adding this excuse here?

Reply:
Yea that would be a better option, but it is currently the bdrv breaking the hard drive install. I would know I have spent 20+ hours checking and testing this...